docs/pages/project/contributing/contributing-error.md

```diff
@@ -9,27 +9,127 @@ category: contributing
 ---
 
 Meshery pervasively uses MeshKit as a golang and service mesh management-specific library in all of its components. MeshKit helps populate error messages with a uniform and useful set of informative attributes. 
+
+This tool analyzes, verifies and updates error codes in Meshery source code trees. It extracts error details into a file that can be used for publishing all error code references on the Meshery website. The objective to create this was to avoid centralized handling of error codes, automating everything
```

Comment:
```suggestion
To help with creating error codes, MeshKit contains a tool that analyzes, verifies and updates error codes in Meshery source code trees. It extracts error details into a file that can be used for publishing all error code references at Meshery [error codes reference page](https://docs.meshery.io/reference/error-codes). The objective to create this was to avoid centralized handling of error codes, automating everything.
```